services/web/server/src/simcore_service_webserver/projects/_handlers_crud.py

```diff
@@ -186,6 +209,57 @@ class _ProjectListParams(BaseModel):
         default=False, description="includes projects marked as hidden in the listing"
     )
 
+    order_by: list[_ProjectOrderBy] = Field(
+        default=None,
+        description="Comma separated list of fields for ordering. The default sorting order is ascending. To specify descending order for a field, users append a 'desc' suffix",
+        example="foo desc, bar",
+    )
+    filters: _ProjectListFilters = Field(
+        default=None,
+        description="Filters to process on the projects list, encoded as JSON",
+        example='{"tags": [1, 5], "classifiers": ["foo", "bar"]}',
+    )
+    search: str = Field(
```

Comment:
the type is incorrect. is pylance not complaining here?

Reply:
fixed